### ISSUE TITLE
Optimize the loading time

### ISSUE THREAD

**Issue by webgnar** [SkateHive/nextskateapp#20]
It is taking way too long to load.

**Comment by hunxjunedo** [comment]
Acknowledgement

**Comment by hunxjunedo** [comment]
So here are some of the things that need to be optimized:

> The media in posts should be lazy-loaded
> The videos should stop playing once out of view
> Something has to be done about the LoaderComponent, it takes as much as 800 MB
> Some sort of compression should be introduced for post media

**Comment by sktbrd** [comment]
all true

**Comment by webgnar** [comment]
@hunxjunedo are you up for the challenge of fixing these issues?

**Comment by hunxjunedo** [comment]
> @hunxjunedo are you up for the challenge of fixing these issues?

Yeah I'm in, but not immediately of course, once I've dealt with the other issue than inshallah more than happy to do that.

**Comment by webgnar** [comment]
what is the other issue you are currently focusing on ?

**Comment by hunxjunedo** [comment]
https://github.com/SkateHive/nextskateapp/issues/17
